metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -109,11 +110,27 @@ class PostgresConfig(BasicSQLAlchemyConfig):
         default=AllowDenyPattern.allow_all(),
         description="Regex patterns for databases to filter in ingestion.",
     )
-    database: Optional[str] = Field(
-        default=None,
-        description="database (catalog). If set to Null, all databases will be considered for ingestion.",
+    ingest_multiple_databases: bool = Field(
+        default=True,
```

Comment:
This will have the effect of automatically enable multi database ingestion even if `database` has been defined in the recipe, which is the opposite of the previous logic.
Just want to call this out in case this is not what you intended.

Reply:
@mayurinehate thoughts on what should be the default? Maybe I'll make it default true only if database isn't provided. Don't want to have too complicated logic though